WebSocketTest/WebSocketTest.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="System.Text.Json" Version="4.6.0" />
```

Comment:
Unnecessary package, .net core 3 has this library pre installed. (same goes for the newtonsoft.json, however ill have to remove that myself sometime)